code/modules/paperwork/photocopier.dm

```diff
@@ -516,6 +520,14 @@
 	else
 		return TRUE
 
+/// Subtype of photocopier that is free to use.
+/obj/machinery/photocopier/gratis
+	desc = "Does the same important paperwork, but it's free to use! The best type of free."
+
+/obj/machinery/photocopier/gratis/setup_components()
+	// it's free! no charge! very cool and gratis-pilled.
+	AddComponent(/datum/component/payment, 0, SSeconomy.get_dep_account(ACCOUNT_CIV), PAYMENT_CLINICAL)
```

Comment:
Does this still reject if they have no ID?

Reply:
![image](https://user-images.githubusercontent.com/34697715/227793439-8662ba3c-59b6-4153-89fb-e7e18d1b34e9.png)

No.